modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -854,20 +854,6 @@ The algorithm normalizes the brightness and increases the contrast of the image.
  */
 GAPI_EXPORTS GMat equalizeHist(const GMat& src);
 
-/** @brief Converts an image from BGR color space to RGB color space.
-
-The function converts an input image from BGR color space to RGB.
-The conventional ranges for B, G, and R channel values are 0 to 255.
-
-Output image is 8-bit unsigned 3-channel image @ref CV_8UC3.
-
-@note Function textual ID is "org.opencv.imgproc.colorconvert.bgr2rgb"
-
-@param src input image: 8-bit unsigned 3-channel image @ref CV_8UC3.
-@sa RGB2BGR
-*/
-GAPI_EXPORTS GMat BGR2RGB(const GMat& src);
-
 //! @} gapi_filters
 
 //! @addtogroup gapi_colorconvert
```

Comment:
Perhaps it is better to move these doxygen headers instead instead of the function.
Keep patches small.

---

How is this change related to this PR? Each PR should have single clear purpose.

Reply:
I noticed that this description accidentally was put into the wrong documentation group and I corrected that. Nothing criminal. This is my code from the previous PR. It seems to me that we shouldn't approach this issue so formally.

Reply:
outdated